.yamllint.yaml

```diff
@@ -0,0 +1,25 @@
+# vim:ts=2:sw=2:et:ai:sts=2
+---
+extends: default
+rules:
+  comments:
+    min-spaces-from-content: 1  # Required by ansible-lint
+  comments-indentation: false  # Required by ansible-lint
+  document-start: disable
+  line-length:
+    max: 80
+  braces:
+    min-spaces-inside: 0  # Required by ansible-lint
+    max-spaces-inside: 1  # Required by ansible-lint
+  octal-values:
+    forbid-implicit-octal: true  # Required by ansible-lint
+    forbid-explicit-octal: true  # Required by ansible-lint
+  quoted-strings:
+    quote-type: single
+    required: only-when-needed
+    extra-required:
+      - ^[0-9:]+$  # IPv6 addresses containing only numbers 0-9 are misparsed
+    extra-allowed:
```

Comment:
Hmm, @NightTsarina:

If I'm not reading wrong, with the current config, yamllint complains quotes are not needed in a line like this one:

```yaml
grafana__smtp__from_address: 'grafana@{{ host_fqdn }}'
```

Do you think we should allow quotes if `{{ .+ }}` is found in the string?

Reply:
It is correct quotes are not needed in that example (for YAML parsing). I am fine with either setting for yamllint

Reply:
Let's keep it simple then (no extra conf.), unless @mrfussion strongly objects.